mobly/controllers/android_device_lib/jsonrpc_client_base.py

```diff
@@ -292,7 +297,17 @@ def _client_receive(self):
         """
         try:
             response = self._client.readline()
-            self.log.debug('Snippet received: %s', response)
+            response_log = str(response)
+            if self.verbose_logging:
+                self.log.debug('Snippet received: %s', response)
+            else:
+                if _MAX_RPC_RESP_LOGGING_LENGTH >= len(response):
```

Comment:
i don't think this check is needed
does any of the unit test fail if you remove this `if` branch?

Reply:
Yes, 2 tests are failed.
For example, if response length is 512, it will display 
`Snippet received: ...... **_-512_** chars are truncated`
because the length of response is less then `_MAX_RPC_RESP_LOGGING_LENGTH`.